8.0.Dockerfile

```diff
@@ -29,6 +29,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=2
+ONBUILD ARG ODOO_WORKERS=2
```

Comment:
I think 4 is a better default here. 2 should be the minimal amount of workers to get a prefork server up and running, but even servers with a single CPU should be able to handle 4 workers nowadays, and it seems a better production default. The devel and test environments override this default already:

https://github.com/Tecnativa/docker-odoo-base/blob/447f57c547896b2212f68db8fa5308da7473bd99/devel.yaml#L36

https://github.com/Tecnativa/docker-odoo-base/blob/447f57c547896b2212f68db8fa5308da7473bd99/test.yaml#L34